### ISSUE TITLE
Bitwarden.sh calls "docker compose" instead of "docker-compose"

### ISSUE THREAD

**Issue by Exportedthought** [bitwarden/server#5043]
### Steps To Reproduce

Attempt to update bitwarden self-hosted
1. Run ./bitwarden.sh updateself
2. Run ./bitwarden.sh update
All as the bitwarden user with permissions required to access docker, per the bitwarden self-host documentation.

### Expected Result

Script execution/update completion.

### Actual Result

Error is received, see below:

docker: 'compose' is not a docker command.
See 'docker --help'

### Additional Context

After manually changing the bitwarden.sh script and the run.sh scripts to use "docker-compose" instead of "docker compose" I was able to get everything to update. It is unclear if this is something self-induced or something wrong with my environment but it seems like the docker compose entries within run.sh/bitwarden.sh should be docker-compose or at least should check as bitwarden.sh seemed to do this previously.

### Githash Version

6862a9bb

### Environment Details

- Operating System: Debian GNU/Linux 12 (bookworm)

### Issue-Link

https://github.com/bitwarden/server/issues/2480

### Issue Tracking Info

- [X] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by Exportedthought** [comment]
Accidentally opened this issue on the unified deployment instead of self-host but I don't see where self-host bugs would be reported.

**Comment by DimebagCFH** [comment]
I have the same issue as well
I'm running on a synology

**Comment by bglf83** [comment]
> I have the same issue as well I'm running on a synology
> 
> @Exportedthought Can you please be a little bit more specific to explain how you fix this into the bitwarden.sh and run.sh ? Thanks in advance, Have a nice day,

I am also using a Synology device.

**Comment by Exportedthought** [comment]
For my situation I simply went through the bitwarden.sh and run.sh scripts and modified any section that referenced "docker compose" to have the dash, as in "docker-compose". I did have to run the ./bitwarden.sh updateself first to get the most recent version of the script and then do the necessary modifications since updateself will replace ./bitwarden.sh. I also had to comment out the function call downloadRunFile in the case statement to ensure it didn't replace those modifications as well. Also note, I ran the update first normally (that's how I ran into issues in the first place) so the modifications that I made are all assuming the latest scripts, I'm not sure all of this is the recommended way to handle this but I'll leave it all here anyways.

**Comment by DimebagCFH** [comment]
Thanks for the update.

**Comment by bglf83** [comment]
Agree, thank you

**Comment by tonyshangrila** [comment]
This helped me get through that November update.  But during December, I realized that my Ubuntu server defaulted to an installation of docker-compose version 1.29.x -- version 2 updates the syntax to exclude the hyphen.  In my case, I used `apt install docker-compose-v2` and now everything works as normal, no editing of scripts required.
Checking your version of `docker-compose` could be the permanent fix you're looking for.
[Additional reference here](https://docs.docker.com/compose/releases/migrate/).
